toolchain/sem_ir/entity_name.h

```diff
@@ -15,22 +15,35 @@ namespace Carbon::SemIR {
 struct EntityName : public Printable<EntityName> {
   auto Print(llvm::raw_ostream& out) const -> void {
     out << "{name: " << name_id << ", parent_scope: " << parent_scope_id
-        << ", index: " << bind_index << "}";
+        << ", index: " << bind_index_value << ", is_template: " << is_template
+        << "}";
   }
 
   friend auto CarbonHashtableEq(const EntityName& lhs, const EntityName& rhs)
       -> bool {
     return std::memcmp(&lhs, &rhs, sizeof(EntityName)) == 0;
   }
 
+  // The index for a compile-time binding. Invalid for a runtime binding, or
+  // for a symbolic binding, like `.Self`, that does not correspond to a generic
+  // parameter (and therefore has no index).
```

Comment:
Sorry, I'm making suggestions even though this is pre-existing.

```suggestion
  // The index for a compile-time binding. Invalid for runtime bindings. Also invalid
  // for a symbolic binding, like `.Self`, that does not correspond to a generic
  // parameter (and therefore has no index).
```

But, invalid or `None`?

Maybe (more changes):

```suggestion
  // The index for a compile-time binding. `None` for runtime bindings, and
  // also for symbolic bindings that don't correspond to a generic parameter
  // (and therefore have no index), such as `.Self`.
```

Reply:
Made even more changes :)